server/routes/incidentio/incidentio_routes.py

```
@@ -0,0 +1,321 @@
+                where += " AND severity = %s"
+                params.append(severity_filter)
+
+            cursor.execute(
+                f"""
+                SELECT id, incident_id, incident_name, incident_status, severity,
+                       incident_type, payload, received_at, created_at
+                FROM incidentio_alerts
+                {where}
+                ORDER BY received_at DESC
+                LIMIT %s OFFSET %s
+                """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)